src/content/docs/de/guides/markdown-content.mdx

```diff
@@ -560,3 +655,22 @@ Wenn du Prism verwenden willst, musst du ein Stylesheet zur Syntaxhervorhebung z
 4. Lade es im [`<head>`-Abschnitt deiner Seite](/de/basics/astro-pages/#seiten-html) mit einem `<link>`-Tag.
 
 Du kannst auch die [Liste der von Prism unterstützten Sprachen](https://prismjs.com/#supported-languages) besuchen, um mehr über die Optionen und deren Verwendung zu erfahren.
+
+## Remote-Markdown abrufen
+
+Astro wurde ursprünglich für lokale Markdown-Dateien, die in deinem Projekt&shy;verzeichnis gespeichert sind, entwickelt. Allerdings gibt es eventuell Situationen, in denen du Markdown von einer Remote-Quelle abrufen musst.Zum Beispiel musst du möglicherweise Markdown von einer entfernten API abrufen und rendern, wenn du deine Website erstellst (oder wenn ein Benutzer eine Anfrage an deine Website stellt, wenn du [SSR](/de/guides/server-side-rendering/) verwendest).
+
+**Astro enthält keine integrierte Unterstützung für entferntes Markdown!** Um entferntes Markdown abzurufen und in HTML zu rendern, musst du deinen eigenen Markdown-Parser aus npm installieren und konfigurieren. Dies **erbt nicht** von den in Astro konfigurierten integrierten Markdown- und MDX-Einstellungen. Stelle sicher, dass du diese Einschränkungen verstehst, bevor du dies in deinem Projekt implementierst.
```

Comment:
```suggestion
**Astro enthält keine integrierte Unterstützung für externes Markdown!** Um externes Markdown abzurufen und in HTML zu rendern, musst du deinen eigenen Markdown-Parser aus npm installieren und konfigurieren. Dies **erbt nicht** von den in Astro konfigurierten integrierten Markdown- und MDX-Einstellungen. Stelle sicher, dass du diese Einschränkungen verstehst, bevor du dies in deinem Projekt implementierst.
```

Bin mir nicht gar nicht sicher, ob `entfernt` ein gängige Übersetzung für `remote` ist. Persönlich würde ich in dem Abschnitt eher `extern` nehmen oder einfach `remote` lassen.

Reply:
passt, dann nehm ich `remote`! wollte es eigentlich eh nicht übersetzen, hab mich dann aber doch dafür entschieden